messages/device-added-notification.json

```diff
@@ -38,7 +38,495 @@
+                    "description": {
+                      "type": "string",
+                      "title": "Property Description",
+                      "description": "Description of the property"
+                    },
+                    "links": {
+                      "type": "array",
+                      "items": {
+                        "type": "object",
+                        "title": "Property Link",
+                        "description": "A property link",
+                        "required": [
+                          "href"
+                        ],
+                        "properties": {
+                          "href": {
+                            "type": "string",
+                            "title": "Link href",
+                            "description": "The href of the link"
+                          },
+                          "rel": {
+                            "type": "string",
+                            "title": "Link rel",
+                            "description": "The type of the relationship"
+                          },
+                          "mediaType": {
+                            "type": "string",
+                            "title": "Link media type",
+                            "description": "The media type of the link"
+                          }
+                        }
+                      }
+                    },
+                    "type": {
+                      "type": "string",
+                      "title": "Property value type",
+                      "description": "The type of the property value",
+                      "enum": [
+                        "null",
+                        "boolean",
+                        "object",
+                        "array",
+                        "number",
+                        "integer",
+                        "string"
+                      ]
+                    },
+                    "unit": {
+                      "type": "string",
+                      "title": "Property unit",
+                      "description": "The unit of the property"
+                    },
+                    "minimum": {
+                      "type": "number",
+                      "title": "Property minimum value",
+                      "description": "The minimum value of the property"
+                    },
+                    "maximum": {
+                      "type": "number",
+                      "title": "Property maximum value",
+                      "description": "The maximum value of the property"
+                    },
+                    "multipleOf": {
+                      "type": "number",
+                      "title": "Property value precision",
+                      "description": "The precision of the value"
+                    },
+                    "enum": {
+                      "type": "array",
+                      "items": {
+                        "type": "string",
```

Comment:
The type will only be `string` if the property type is also `string`.

Reply:
This also applies to the actions and events.

Reply:
Done